lib/rules/dom-elements-style-is-object.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @fileoverview Enforce style prop value being an object and havng sting values
+ * @author David Petersen
+ */
+'use strict';
+
+var variableUtil = require('../util/variable');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Enforce style prop value being an object and havng sting values',
```

Comment:
What should the rule for validating the style object be named?  `style-prop-is-valid`?